packages/console/src/index.ts

```diff
@@ -49,11 +39,11 @@ export {
   info,
   assert,
   debug,
-  logError,
-  logWarn,
-  logWarnDeprecated,
-  logInfo,
-  logDebug,
-  logAssert,
-  consoleLog
+  /*#__PURE__*/ error as logError,
+  /*#__PURE__*/ warn as logWarn,
+  /*#__PURE__*/ warnDeprecated as logWarnDeprecated,
+  /*#__PURE__*/ info as logInfo,
+  /*#__PURE__*/ debug as logDebug,
+  /*#__PURE__*/ assert as logAssert,
+  /*#__PURE__*/ log as consoleLog
```

Comment:
as I could tell the only thing the plugin did is to insert these PURE comments (in a wrong way because it did it twice)